src/datagen/generated/domum_ornamentum/data/minecraft/tags/blocks/mineable/shovel.json

```diff
@@ -0,0 +1,21 @@
+{
+  "values": [
+    "domum_ornamentum:black_brick_extra",
+    "domum_ornamentum:blue_brick_extra",
+    "domum_ornamentum:brown_brick_extra",
+    "domum_ornamentum:brick_extra",
+    "domum_ornamentum:cyan_brick_extra",
+    "domum_ornamentum:gray_brick_extra",
+    "domum_ornamentum:green_brick_extra",
+    "domum_ornamentum:light_blue_brick_extra",
```

Comment:
shovel?

Reply:
Oh wait that should be pickaxe, I'm going to rename the source from "clay" to "brick'